src/hotspot/share/utilities/ostream.cpp

```diff
@@ -186,18 +186,9 @@ void outputStream::put(char ch) {
   write(buf, 1);
 }
 
-#define SP_USE_TABS false
-
 void outputStream::sp(int count) {
   if (count < 0)  return;
```

Comment:
Is this  needed?

Reply:
The `count < 0` check? Yes, as Shipilev mentioned:

>Looks okay. I thought what would happen if _indentation < _position, but sp() seems to handle count < 0 fine.

So it's nice that we don't have to care at the call site.

Reply:
Ah, okay. Missed that.